src/main/resources/application.conf

```diff
@@ -7,21 +7,9 @@ url {
 client {
     id = "tdr-backend-checks"
     id = ${?CLIENT_ID}
-    secret = "secretplaceholder"
-    secret = ${?CLIENT_SECRET}
+    secret_path = "/intg/keycloak/backend_checks_client/secret"
```

Comment:
Should this be pointing to the intg secret as a default?

Reply:
It doesn't matter greatly but I've put it back to secretplaceholder.